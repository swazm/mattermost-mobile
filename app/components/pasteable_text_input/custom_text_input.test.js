// Copyright (c) 2015-present SWAZM, Inc. All Rights Reserved.
// See LICENSE.txt for license information.
import React from 'react';
import {shallow} from 'enzyme';

import CustomTextInput from './custom_text_input';

describe('CustomTextInput', () => {
    test('should render custom text input', () => {
        const onPaste = jest.fn();
        const text = 'My Text';
        const component = shallow(
            <CustomTextInput onPaste={onPaste}>{text}</CustomTextInput>
        );
        expect(component).toMatchSnapshot();
    });
});
