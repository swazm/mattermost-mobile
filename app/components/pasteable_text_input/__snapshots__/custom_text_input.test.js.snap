// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTextInput should render custom text input 1`] = `
<TextInput
  allowFontScaling={true}
  onPaste={[MockFunction]}
  rejectResponderTermination={true}
  underlineColorAndroid="transparent"
>
  My Text
</TextInput>
`;
